osu.Game/Overlays/Profile/Header/Components/PreviousUsernames.cs

```diff
@@ -84,6 +86,7 @@ public PreviousUsernames()
                                 RelativeSizeAxes = Axes.X,
                                 AutoSizeAxes = Axes.Y,
                                 Direction = FillDirection.Full,
+                                AlwaysPresent = true,
```

Comment:
Why? Every `AlwaysPresent` usage should ideally come with explanation. Ideally as an inline comment.